website/source/docs/secrets/ad/index.html.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: "docs"
+page_title: "Active Directory - Secrets Engines"
+sidebar_current: "docs-secrets-active-directory"
+description: |-
+  The Active Directory secrets engine for Vault generates passwords dynamically based on
+  roles.
+---
+
+# Active Directory Secrets Engine
+
+The Active Directory (AD) secrets engine is a plugin residing [here](https://github.com/hashicorp/vault-plugin-secrets-active-directory).
+
+The AD secrets engine rotates AD passwords dynamically,
+and is designed for a high-load environment where many instances may be accessing
+a shared password simultaneously. With a simple set up and a simple creds API,
+it doesn't require instances to be manually registered in advance to gain access. 
+As long as access has been granted to the creds path via a method like 
+[AppRole](https://www.vaultproject.io/api/auth/approle/index.html), they're available.
+
+Passwords are lazily rotated based on preset TTLs and can have a length configured to meet 
```

Comment:
You may want to elaborate a little more on the password rotation process. The preceding paragraph say they're rotated dynamically, and this paragraph says "lazily... based on preset TTLs".  It might not be clear what is actually triggering a rotation and when.

Reply:
Ah, yes, you're right, that's a very important behavior. I will.

Reply:
Also added this on a later line.